todo_app/flask_config.py

```diff
@@ -11,6 +11,8 @@ def __init__(self):
         self.github_app_id = os.getenv('OAUTH_APP_ID')
         self.github_app_secret = os.getenv('OAUTH_APP_SECRET')
         self.LOGIN_DISABLED = os.getenv('LOGIN_DISABLED')
+        self.LOG_LEVEL = os.getenv('LOG_LEVEL')
+        self.LOGGLY_TOKEN = os.getenv('LOGGLY_TOKEN')
```

Comment:
Looks like you're working orthogonally to M12 so it makes sense but worth bearing in mind that when you merge both it would be good to add this variable to your M12 setup (Terraform, set it in the pipeline etc.) so that you Azure app is actually logging to Loggly